.github/workflows/ci.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
The commit message tells about moving from NodeX to NodeY but we don't have node dependencies. The code moves from checkout@v2 to v3 and from setup-go@v2 to v3. Can you explain the change in the commit message and link to the relevant docs?

The issue contains a link to the docs, but no other text explaining the issue and the needed changes.